src/test/javascript/spec/component/course/course-exercises.component.spec.ts

```diff
@@ -197,31 +189,31 @@ describe('CourseExercisesComponent', () => {
         ];
 
         // Number of exercises in the course must be 4, since we have added 4 exercises
-        expect(component.course!.exercises!.length).to.equal(4);
+        expect(component.course!.exercises!.length).toEqual(4);
```

Comment:
Same as above. Use `toHaveLength()`
```suggestion
        expect(component.course!.exercises!).toHaveLength(4);
```